usaspending_api/decorators.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+
+logger = logging.getLogger('console')
+
+
+def deprecated(function):
+    """Add deprecation warning to endpoint"""
+    def wrap(request, *args, **kwargs):
+        response = function(request, *args, **kwargs)
+        logger.warning('Endpoint "{}" is deprecated. Please move to v2 endpoint.'.format(request.path))
```

Comment:
Are vs endpoints always `v2/<v1endpoint>`? Otherwise we might want to add the url of the new function as a second parameter.

Reply:
Not all v1 endpoints have a complementary v2 version

Reply:
With the less specific message you suggested this works better, but a reference to the v2 endpoint list might be an improvement. Future implementers might only know about our API based on legacy systems' calls to it.